backend/client_api.c

```diff
@@ -20,6 +20,77 @@
 
 #include "client_api.h"
 
+struct dbc_stat dbc_stats;
+
+void zeroize_dbc_ops_stats(struct dbc_ops_stat *ops_stat, const char *name) {
+	ops_stat->name = name;
+	ops_stat->called = 0;
```

Comment:
Add `completed` counter that counts how many completed operations we have. It should be equal to `called` or the difference are currently outstanding operations. Success + error should equal `completed`.